CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,4 @@
+Code of conduct
+===============
+
+This repository is governed by Mozilla's code of conduct and etiquette guidelines. For more details please see the [Mozilla Community Participation Guidelines] (https://www.mozilla.org/en-US/about/governance/policies/participation/) and [Developer Etiquette Guidelines] (https://bugzilla.mozilla.org/page.cgi?id=etiquette.html).
```

Comment:
As mentioned in https://github.com/mozilla/addons-linter/pull/1045, can we remove `en-US` from here?